src/test/librbd/journal/test_Entries.cc

```diff
@@ -111,10 +111,21 @@ class TestJournalEntries : public TestFixture {
     return true;
   }
 
+  bool is_rbd_rwl_enabled() {
+#if defined(WITH_RBD_RWL)
+    std::string value;
+    EXPECT_EQ(0, _rados.conf_get("rbd_rwl_enabled", value));
+    return value == "true";
+#else
+    return false;
+#endif
+  }
+
 };
 
 TEST_F(TestJournalEntries, AioWrite) {
   REQUIRE_FEATURE(RBD_FEATURE_JOURNALING);
+  REQUIRE(!is_rbd_rwl_enabled());
```

Comment:
Nit: what's the background on this? To me it's not entirely obvious why the test is disabled in RWL mode.

Reply:
Currently, I disabled RWL cache when journaling is enabled. As I think journaling/mirror doesn't really work when data is cached in RWL.

Reply:
Any particial reason why that's true?

Reply:
In fact, RWL cache can co-exist with journaling. But when co-existing with journaling, the journal is recorded only when data is flushed to OSD.    
On the other side, the metadata set/remove also impact the current journaling test cases.